.gitignore

```diff
@@ -1,27 +0,0 @@
-# Logs
```

Comment:
I removed this file originally thinking it didn't make much sense in the `gh-pages` branch, but given how we're now using `master` and feature branches, I'm not sure that was a good idea. :(